modules/ztp-sno-du-setting-rcu-normal.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-setting-rcu-normal_{context}"]
+= Setting rcu_normal
+
+The following `MachineConfig` CR configures the system to set `rcu_normal` to 1 after the system has finished startup. This improves kernel latency for vDU applications.
```

Comment:
```suggestion
The following `MachineConfig` custom resource (CR) configures the system to set `rcu_normal` to 1 after the system has finished startup. This improves kernel latency for vDU applications.
```
Do you need to explain the "1" value? Not sure if it's a state or count and if your readers should know.

Reply:
I personally don't think we need to get into it, it is detailed in the linked in 08-set-rcu-normal-master.yaml which I think is enough as it gives a description of what it does.